azerothcore/azerothcore-wotlk
# feat(Core/MapUpdate): switch from ACE_Method_Request to PCQ

## CHANGES PROPOSED:
- Switch from ACE_Method_Request to PCQ
- Detele `DelayExecutor`

## ISSUES ADDRESSED:
- Part #3451 

## TESTS PERFORMED:
- Build and in-game

## HOW TO TEST THE CHANGES:
- Check core performance when updating maps (enter in the world and wait ~1-2 min, change area)
- Check work `LFG` system

## Target branch(es):

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
